optimizely/event/event_processor.py

```diff
@@ -266,3 +266,47 @@ def stop(self):
 
     if self.is_running:
       self.logger.error('Timeout exceeded while attempting to close for ' + str(self.timeout_interval) + ' ms.')
+
+
+class ForwardingEventProcessor(BaseEventProcessor):
+  """
+  ForwardingEventProcessor serves as the default EventProcessor.
+
+  The ForwardingEventProcessor sends the LogEvent to EventDispatcher as soon as it is received.
+  """
+
+  def __init__(self, event_dispatcher, logger, notification_center=None):
```

Comment:
`logger` should be optional and we should set it to `NoOpLogger` if not provided.